plugins/git/README.md

```diff
@@ -154,15 +156,14 @@ plugins=(... git)
 | gsr                  | git svn rebase                                                                                                                   |
 | gss                  | git status -s                                                                                                                    |
 | gst                  | git status                                                                                                                       |
-| gsta                 | git stash push                                                                                                                   |
-| gsta                 | git stash save                                                                                                                   |
+| gsta                 | git stash push / git stash save                                                                                                  |
```

Comment:
`gsta` has different meanings depending on the git version. I find it easier to understand if theere is only one entry for `gsta` instead of two